unity/Assets/Maroon/scenes/experiments/OpticsSimulations/Scripts/LaserSelectionHandler.cs

```diff
@@ -0,0 +1,245 @@
+﻿//
+//Author: Tobias Stöckl
+//
+using UnityEngine;
+using Maroon.Physics;
+using Maroon.UI;
+using GEAR.Localization;
+
+public class LaserSelectionHandler : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [HideInInspector]
+    public GameObject CurrActiveLaser;
+    public GameObject LaserControlPanel;
+    public GameObject LaserPointerPrefab;
+
+    public QuantityFloat ActiveLaserIntensity;
+    public QuantityFloat ActiveLaserWavelength;
+
+    public QuantityFloat ActiveLaserRefractiveIndex;
+
+    private int _numLaserPointers = 0;
+    public int MaxLasers = 100;
+
+
+    public bool OnUIpanel { get; set; } = false;
+
+
+    private Vector3 _laserArrayStartPoint;
```

Comment:
maybe change this to
[SerializeField] private Transform 
this would allow us to assign this in the Unity Inspector.